class-wc-calypso-bridge-shared.php

```diff
@@ -133,6 +133,21 @@ public function add_scripts() {
 
 		wp_enqueue_script( 'wc-calypso-bridge' );
 		wp_enqueue_style( 'wc-calypso-bridge' );
+
+		// Inject the WC data store patch for WooCommerce <= 7.6 with Gutenberg 15.5+
+		$has_gutenberg             = is_plugin_active( 'gutenberg/gutenberg.php' );
+		$gutenberg_version         = $has_gutenberg ? get_plugin_data( WP_PLUGIN_DIR . '/gutenberg/gutenberg.php' )['Version'] : false;
+
+		if ( 
+			defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '7.6.0', '<=' ) &&
```

Comment:
```suggestion
			defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '7.7.0', '<' ) &&
```

I think we'll make the Core fix only in WC 7.7 now. What do you think about changing it this?

Reply:
Missed that. Sounds good, thanks!